docs/UserGuide.md

```diff
@@ -283,7 +283,8 @@ Showcases a note.
 
 Format: `shownote INDEX​`
 
-* Shows the note at the specified `INDEX`. The index refers to the index number shown in the displayed note list. The index **must be a positive integer** 1, 2, 3, …​
+* Shows the dictionary content at the specified `INDEX`.
+* The index refers to the index number shown in the displayed Note list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Is this description for `shownote` or `showdc`?

Reply:
fix